src/site/assets/robots.txt

```diff
@@ -1,90 +1,3 @@
-User-agent: usasearch
-Disallow: /
-Allow: /about_va
-Allow: /accountability
-Allow: /adr
-Allow: /ADVISORY
-Allow: /AssetManagement
-Allow: /bluebutton
-Allow: /budget
-Allow: /CENTERFORMINORITYVETERANS
-Allow: /CFBNPARTNERSHIPS
-Allow: /communitycare
-Allow: /debtman
-Allow: /diagnosticem
-Allow: /employee
-Allow: /finance
-Allow: /fromwartohome
-Allow: /fund
-Allow: /GERIATRICS
-Allow: /grecc
-Allow: /gulfwaradvisorycommittee
-Allow: /health
-Allow: /health/ISP
-Allow: /health/medicalinspector
-Allow: /health/patientadvocate
-Allow: /healthbenefits
-Allow: /HEALTHBENEFITS/vtp
-Allow: /healthequity
-Allow: /healthpolicyplanning
-Allow: /healthresourcecenter
-Allow: /homeless
-Allow: /homeless/dchv.asp
-Allow: /homeless/dental.asp
-Allow: /ICARE
-Allow: /identitytheft
-Allow: /imaging
-Allow: /jobs
-Allow: /library
-Allow: /lmr
-Allow: /nac
-Allow: /NCOD
-Allow: /nursing
-Allow: /oaa
-Allow: /oaem
-Allow: /oca
-Allow: /oei
-Allow: /ofcadmin
-Allow: /ogc
-Allow: /ogc/apps/accreditation
-Allow: /ohrm
-Allow: /oig
-Allow: /OM
-Allow: /opa
-Allow: /OPAL
-Allow: /open
-Allow: /OPTOMETRY
-Allow: /orm
-Allow: /oro
-Allow: /orpm
-Allow: /osdbu
-Allow: /painmanagement
-Allow: /patientcenteredcare
-Allow: /PBI
-Allow: /performance
-Allow: /PIVproject
-Allow: /plo
-Allow: /qualityofcare
-Allow: /rac-gwvi
-Allow: /recovery
-Allow: /samedayservices
-Allow: /securityinvestigationscenter
-Allow: /survivors
-Allow: /tribalgovernment
-Allow: /trm
-Allow: /VADODHEALTH
-Allow: /vaforms
-Allow: /valnet
-Allow: /vapubs
-Allow: /vastorenac
-Allow: /vetdata
-Allow: /vetsinworkplace
-Allow: /vhaemergencymanagement
-Allow: /vhapublications
-Allow: /vler
-Allow: /vso
-Allow: /womenvet
-
```

Comment:
Here it looks like we were defaulting to disallow every directory (to prevent crawling of private directories, I assume), then explicitly listing directories that are allowed. That list either needs to be updated (to add every local top-level directory, which we'll probably never remember to keep updated), or we should deleted the whole thing. Removing it seems like the easier choice.

Reply:
Do we have to remove it? I'm a little hesitant since I don't understand why we did this in the first place.

Reply:
I think it was copied over from the pre-WBC VA.gov - @rianfowler do you happen to know?

I'm just wondering if this is negatively impacting our search.gov search engine, since to me it's suggesting that pages under `/health-care` should not be indexed since there isn't a `Allow: /health-care`

Reply:
I believe these are user agent specific directives (`usasearch` I think is for search.va.gov's crawler. I'm not sure who set these up). I'm a little confused by this diff because I don't see these `usasearch` directives in master.

Reply:
well I guess it would help if I actually looked at master. I think we should leave these and I think @BBarone1 knows who is responsible for search.va.gov's specific need for this to be in the robots.txt.

Reply:
I think that's search.gov's user agent, which we're still using, so I'm thinking this is hurting our own search engine. But I'm not sure. Either way, I reverted that commit 👌

Reply:
Moving this convo to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/17579